products/health-care/appointments/va-online-scheduling/research/may-2021-facilities-personalization-research/conversation-guide.md

```diff
@@ -114,27 +114,27 @@ Scenario 1 - opt-out flow (50% of participants)
 
 [Prototype - opt-out flow](https://adhoc.invisionapp.com/console/share/BX2JRZTUR2)
 
-You recently moved to a new city, Evanston Illinois. You're interested in setting up a follow-up visit with your primary care doctor there.
+*Last year*, you moved to a new city, Evanston Illinois _from Seattle_. You're interested in setting up a follow-up visit with your primary care doctor there. ```comment: see suggested addition of 'last year' and  'Seattle.' In this question you may need to establish that they have already been to a primary care doctor. My first read on this was ' if i just moved, i won't have a doctor there.' Additionally since they will see Evanston and Seattle in this list, you should set that up for them.```
 
 ### Task 1 - review facility list in VAOS
 
-*Start on VA.gov home page*
+*Start on VA.gov home page* ```comment: what is your plan if they do not go to VAOS?```
```

Comment:
If they don't know where to go I'll suss out where they expected to go, then direct them to the VAOS link. Might be worth adding an explicit q here to call that out, I'll revisit.